src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/Value.tsx

```diff
@@ -15,7 +15,7 @@ import SafeEtherscanLink from 'src/components/EtherscanLink'
 const useStyles = makeStyles(styles)
 
 const NestedWrapper = styled.div`
-  text-indent: 24px;
+  word-break: break-word;
```

Comment:
By removing `text-indent` values won't be properly aligned.

On the other hand, by setting these word breakers attributes at the column level, seems to do the trick:

```css
.column {
  word-break: break-word;
  white-space: normal;
}
```

![break-word-column](https://user-images.githubusercontent.com/3315606/92020245-3ca5d180-ed2e-11ea-976a-9e23e5ad609e.gif)

Reply:
this is how I see it:
![image](https://user-images.githubusercontent.com/4376509/92022088-1fbecd80-ed31-11ea-8bf5-1455cd4fdd04.png)

Reply:
the div wrapping the method name and value already has a text-ident. 
![image](https://user-images.githubusercontent.com/4376509/92022309-71675800-ed31-11ea-8ccb-51af802e0f5c.png)

Reply:
but that `NestedWrapper` is used for those cases where an argument is an array of params, not listed here.